components/typography/Base/Ellipsis.tsx

```diff
@@ -134,18 +102,15 @@ export default function EllipsisMeasure(props: EllipsisProps) {
 
   // ========================= Full Content =========================
   // Used for measure only, which means it's always render as no need ellipsis
-  const fullContent = React.useMemo(() => children(nodeList, false), [text]);
+  const fullContent = React.useMemo(() => children(nodeList, false), [text, children, nodeList]);
 
   // ========================= Cut Content ==========================
   const [ellipsisCutIndex, setEllipsisCutIndex] = React.useState<[number, number] | null>(null);
   const cutMidRef = React.useRef<MeasureTextRef>(null);
 
   // ========================= NeedEllipsis =========================
   const measureWhiteSpaceRef = React.useRef<HTMLElement>(null);
-
   const needEllipsisRef = React.useRef<MeasureTextRef>(null);
-
-  // Measure for `rows-1` height, to avoid operation exceed the line height
   const descRowsEllipsisRef = React.useRef<MeasureTextRef>(null);
```

Comment:
```suggestion
  // Measure for `rows-1` height, to avoid operation exceed the line height
  const descRowsEllipsisRef = React.useRef<MeasureTextRef>(null);
```